lib/ae_mdw/db/async_store.ex

```diff
@@ -0,0 +1,89 @@
+defmodule AeMdw.Db.AsyncStore do
+  @moduledoc """
+  Implementation of Store protocol to be used by asynchronous tasks.
+
+  Its operations have immediate in-memory effect (not persisted) and
+  is backed by a cache with TTL of 1 day.
+  """
+
+  alias AeMdw.Database
+  alias AeMdw.EtsCache
+
+  @derive AeMdw.Db.Store
+  defstruct tid: nil
```

Comment:
`:tid` is never nil, or a possible valid value defined by the spec, so we should use `defstruct [:tid]` here instead

Reply:
when a value is not specified, it assigns nil (same as `defstruct [:tid]`)